hypervisor/include/common/irq.h

```diff
@@ -31,10 +35,44 @@ struct irq_desc {
 #endif
 };
 
+
+/**
+ * @brief Request an interrupt
+ *
+ * Request interrupt num if not specified, and register irq action for the
+ * specified/allocated irq.
+ *
+ * @param[in]	req_irq	irq_num to request, if IRQ_INVALID, a free irq
+ *		number will be allocated
+ * @param[in]	action_fn	Function to be called when the IRQ occurs
+ * @param[in]	priv_data 	Private data for action function.
+ * @param[in]	flags	Interrupt type flags, including:
+ *			IRQF_NONE;
+ *			IRQF_LEVEL - 1: level trigger; 0: edge trigger;
+ *    			IRQF_PT    - 1: for passthrough dev
+ *
+ * @return valid irq num - on success
+ * @return IRQ_INVALID - on failure
+ */
 int32_t request_irq(uint32_t req_irq, irq_action_t action_fn, void *priv_data,
 			uint32_t flags);
 
+/**
+ * @brief Free an interrupt
+ *
+ * Free irq num and unregister the irq action.
+ *
+ * @param[in]	irq	irq_num to be freed
+ */
 void free_irq(uint32_t irq);
 
+/**
+ * @brief Set interrupt trigger mode
+ *
+ * Set the irq trigger mode: edge-triggered or level-triggered
+ *
+ * @param[in]	irq	irq_num of interupt to be set
+ * @param[in]	is_level_triggered	Trigger mode to set
```

Comment:
You have a mismatch here, `is_level_trigger` vs. `is_level_triggered`.
```suggestion
 * @param[in]	is_level_trigger	Trigger mode to set
```

Reply:
Note that updating all code to use `is_level_triggered` instead of `is_level_trigger` would improve readability, so I would recommend you do that instead of just fixing the mismatch.

Reply:
Yes, I updated the code to use is_level_triggered.
Thanks.